packages/services/functions.json

```diff
@@ -17,7 +17,7 @@
             "scriptFile": "../dist/src/functions/azure/syncservice.js"
         }
     },
-    {   
+    {
         "name": "statusservice",
         "template": {
             "disabled": false,
```

Comment:
Do we need this file anymore?

Reply:
this file is the one used to build the function.json for each function. Do you want me to delete it?

Reply:
deleted!